mautic/mautic
# Add back the Tracking Page Enabled setting for OneSignal

[//]: # ( Please answer the following questions: )

| Q  | A
| --- | ---
| Bug fix? | Y
| New feature? | 
| Related user documentation PR URL | 
| Related developer documentation PR URL | 
| Issues addressed (#s or URLs) | 
| BC breaks? | 
| Deprecations? | 

[//]: # ( Note that all new features should have a related user and/or developer documentation PR in their respective repositories. )

[//]: # ( Required: )
#### Description:

2.8.0 removed the `tracking_page_enabled` configuration option from the OneSignal configuration. This PR adds it back.

This PR also adds the ability for devs to specify tooltips on the supported features within their integrations in order to add better contextual help.

![screen shot 2017-05-16 at 5 29 16 pm](https://cloud.githubusercontent.com/assets/718028/26129178/3ad1a992-3a5d-11e7-9d91-6cfc3aa51eed.png)

To utilize the tooltips on the Supported Features section, devs must add a `getSupportedFeatureTooltips` method to their integration class, as shown here: https://github.com/mautic/mautic/pull/4077/files#diff-6e6961738168df27d8d8933da80cae18R55

[//]: # ( As applicable: )
#### Steps to reproduce the bug:
1. Open OneSignal integration settings
2. See `tracking page enabled` is missing from Feature Settings section

#### Steps to test this PR:
1. Apply PR
2. Check feature settings - the `tracking page enabled` setting is back, AND has a tooltip!
